lib/pbench/agent/utils.py

```diff
@@ -397,12 +396,12 @@ class Return(NamedTuple):
         stdout: str
         stderr: str
 
-    def __init__(self, ssh_cmd: Path, ssh_args: List[str], cmd: str):
+    def __init__(self, ssh_cmd: str, ssh_args: List[str], cmd: str):
         """
         Create an SSH template object
 
         Args:
-            ssh_cmd: Path to the ssh command
+            ssh_cmd: file path of the ssh command
```

Comment:
`Path`/"path" ... sigh. Could be misleading, but hard to clarify, and at least the type hint is now consistent with what we expect. 🍤